Vagrant/laeVagrantfile

```diff
@@ -39,14 +39,12 @@ Vagrant.configure(2) do |config|
       rails_env: "production",
       application_host: 'localhost:8484',
       passenger_server_name: 'localhost',
-      figgy_host_name: 'localhost',
```

Comment:
hey @escowles does an lae Vagrantfile need figgy references?

Reply:
@kayiwa Yep! It pulls JSON-LD and IIIF manifests from Figgy to do a full reindex.

Reply:
thanks!